mitxpro/utils.py

```diff
@@ -485,6 +485,40 @@ def make_csv_http_response(*, csv_rows, filename):
     return response
 
 
+def make_csv_http_response_with_instructions(*, csv_rows, filename):
+    """
+    Create a HttpResponse for a CSV file.
+
+    Args:
+        csv_rows (iterable of dict): An iterable of dict, to be written to the CSV file
+        filename (str): The filename to suggest for download
```

Comment:
Given that this method mirrors `make_csv_http_response` with instructions added, please include an iterable for `instructions` separately.